api/v1alpha1/constants.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2023 D2iQ, Inc. All rights reserved.
+// Copyright 2024 D2iQ, Inc. All rights reserved.
```

Comment:
nit: would advise running this by legal first.

Reply:
@jimmidyson is planning to change it later. changing year is a standard practice.